antlr-kotlin-examples-jvm/gradle.properties

```diff
@@ -1,5 +0,0 @@
-org.gradle.jvmargs=-Xmx8192m
```

Comment:
don't we need more memory to build this?

Reply:
I just removed all old gradle project files.. it is still compiling. I think it is not required.